src/tool_cfgable.h

```diff
@@ -298,6 +298,7 @@ struct OperationConfig {
   struct State state;             /* for create_transfer() */
   bool rm_partial;                /* on error, remove partially written output
                                      files */
+  bool mptcp;                     /* enable MPTCP support */
```

Comment:
can you move this a bit above, just between `file_clobber_mode;` and `struct GlobalConfig *global;`.

Because `struct OperationConfig *next;   /* Always last in the struct */` just above